static/app/utils/formatters.tsx

```diff
@@ -148,70 +148,60 @@ export function getDuration(
 }
 
 /**
- * Returns a human readable exact duration.
+ * Translates seconds into human readable format of seconds, minutes, hours, days, and years
  *
- * e.g. 1 hour 25 minutes 15 seconds
+ * NOTE: seconds input can be a decimal
+ * @param  {number} seconds The number of seconds to be processed
+ * @return {string}         The phrase describing the amount of time
```

Comment:
No need for the `{number}` `{string}` these are on the types already
```suggestion
 * @param  seconds The number of seconds to be processed
 * @return         The phrase describing the amount of time
```

Reply:
You're also not documenting `minDuration`